### PR TITLE
Update release-team.md

### PR DESCRIPTION
**Pull request by Damans227**
Updated the correct names. Btw, I am shadowing on release notes team, Hi!

### CODE REVIEW

**Review by JamesLaverack — APPROVED** [review]
/lgtm
/approve
This was totally my bad. Great spot @Damans227 !

File: releases/release-1.22/release-team.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
...
-| Enhancements | James Laverack ([@jameslaverack](https://github.com/jameslaverack) / Slack: `@james.laverack`)| |
-| CI Signal | Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`) | Xander Grzywinski ([@salaxander](https://github.com/salaxander) / Slack: `@Xander`), Rey Lejano ([@reylejano](https://github.com/reylejano) / Slack: `@rlejano`), Grace Nguyen ([@gracenng](https://github.com/gracenng) / Slack: `@Grace Nguyen`), Supriya Premkumar ([@supriya-premkumar](https://github.com/supriya-premkumar) / Slack: `@supriya premkumar`), Joseph Sandoval ([@jrsapi](https://github.com/jrsapi) / Slack: `@Joseph`) |
+| Enhancements | James Laverack ([@jameslaverack](https://github.com/jameslaverack) / Slack: `@james.laverack`)| | Xander Grzywinski (@salaxander / Slack: @Xander), Rey Lejano (@reylejano / Slack: @rlejano), Grace Nguyen (@gracenng / Slack: @Grace Nguyen), Supriya Premkumar (@supriya-premkumar / Slack: @supriya premkumar), Joseph Sandoval (@jrsapi / Slack: @Joseph)
```
Comment (JamesLaverack) [inline releases/release-1.22/release-team.md:6]:
```suggestion
| Enhancements | James Laverack ([@jameslaverack](https://github.com/jameslaverack) / Slack: `@james.laverack`)| Xander Grzywinski (@salaxander / Slack: @Xander), Rey Lejano (@reylejano / Slack: @rlejano), Grace Nguyen (@gracenng / Slack: @Grace Nguyen), Supriya Premkumar (@supriya-premkumar / Slack: @supriya premkumar), Joseph Sandoval (@jrsapi / Slack: @Joseph)
```

**Review by JamesLaverack — CHANGES_REQUESTED** [review]

File: releases/release-1.22/release-team.md [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
-| Enhancements | James Laverack ([@jameslaverack](https://github.com/jameslaverack) / Slack: `@james.laverack`)| |
-| CI Signal | Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`) | Xander Grzywinski ([@salaxander](https://github.com/salaxander) / Slack: `@Xander`), Rey Lejano ([@reylejano](https://github.com/reylejano) / Slack: `@rlejano`), Grace Nguyen ([@gracenng](https://github.com/gracenng) / Slack: `@Grace Nguyen`), Supriya Premkumar ([@supriya-premkumar](https://github.com/supriya-premkumar) / Slack: `@supriya premkumar`), Joseph Sandoval ([@jrsapi](https://github.com/jrsapi) / Slack: `@Joseph`) |
+| Enhancements | James Laverack ([@jameslaverack](https://github.com/jameslaverack) / Slack: `@james.laverack`)| Xander Grzywinski (@salaxander / Slack: @Xander), Rey Lejano (@reylejano / Slack: @rlejano), Grace Nguyen (@gracenng / Slack: @Grace Nguyen), Supriya Premkumar (@supriya-premkumar / Slack: @supriya premkumar), Joseph Sandoval (@jrsapi / Slack: @Joseph)
+| CI Signal | Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`) |
```
Comment (JamesLaverack) [inline releases/release-1.22/release-team.md:7]: Just need to preserve the links on the moved text, like so:

```suggestion
| Enhancements | James Laverack ([@jameslaverack](https://github.com/jameslaverack) / Slack: `@james.laverack`)| Xander Grzywinski ([@salaxander](https://github.com/salaxander) / Slack: `@Xander`), Rey Lejano ([@reylejano](https://github.com/reylejano) / Slack: `@rlejano`), Grace Nguyen ([@gracenng](https://github.com/gracenng) / Slack: `@Grace Nguyen`), Supriya Premkumar ([@supriya-premkumar](https://github.com/supriya-premkumar) / Slack: `@supriya premkumar`), Joseph Sandoval ([@jrsapi](https://github.com/jrsapi) / Slack: `@Joseph`) |
| CI Signal | Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`) | |
```

**Review by savitharaghunathan — APPROVED** [review]
/lgtm

**Comment by pmmalinov01** [comment]
Great spot @Damans227 
/party

**Comment by sladyn98** [comment]
This is great @Damans227 ! Thanks

**Pull request merged**